### ISSUE TITLE
Allow enforce minimum password score

### ISSUE THREAD

**Issue by DumpName** [0x3333/smbpasswd-web#8]
Hi,<br>
first of all I have to say that this project saved me a lot of trouble trying to find a way for my samba only users to change their passwords.<br>
I just discovered that the password scoring mechanism might need some improvements.
It would be great if a password could only be submitted if it meets some criteria like the ones set by samba by default.<br>
Otherwise a password could be submitted and the user is shown an error if his password does not meet the samba policy.<br>
Maybe some configurable solution would work here.<br>
At the moment I modified the app.js file that the standard policy has to be fullfilled.<br>
I don't know if you have the time to add something like this at the moment, therefor I can only offer to contribute in the next couple of weeks.<br>
Regards

**Comment by 0x3333** [comment]
Nice to hear it helps!

I'd like to have some kind of configurable score mechanism to allow the user to create their own score, but it is not easy to plugin nor I have time for that now, but enforcing a minimum score is pretty straightforward. I'll check how to enforce a minimum score.

**Comment by DumpName** [comment]
As I just had to reconfigure my servers I stumbled over my local installation of this project. While scrlling through the git I rediscovered my issue.
In one of my projects I just deployed zxcvbn as a password strength meter. Maybe this would be a nice addition in here as well.
I have not done anything in python before. Otherwise I would like to contribute myself. However at the moment python-code feels to much like whitchcraft for comfort ;)
